doc/arch/015-online-patient-lookup.md

```diff
@@ -0,0 +1,121 @@
+# Online Patient Lookup
+
+## Status
+
+Proposed on 2020-04-12
+
+## Context
+
+Want want to ensure that a patient’s medical records within the state
```

Comment:
I'm guessing this should say "we want".

Reply:
haha, the mind just glosses over such typos 😂  thanks, will fix

Reply:
Fixed.